voter-dapp/src/ActiveRequests.js

```diff
@@ -90,18 +90,30 @@ function ActiveRequests({ votingAccount, votingGateway }) {
 
   // Only display non-blacklisted price requests (uniquely identifier by identifier name and timestamp)
   const [showSpamRequests, setShowSpamRequests] = useState(false);
+  const [hasSpamRequests, setHasSpamRequests] = useState(false);
   const [pendingRequests, setPendingRequests] = useState([]);
   useEffect(() => {
+    setHasSpamRequests(false);
+
     if (allPendingRequests) {
-      const _pendingRequests = allPendingRequests.filter(req => {
-        if (showSpamRequests || !REQUEST_BLACKLIST[hexToUtf8(req.identifier)]) return true;
+      const _nonBlacklistedRequests = allPendingRequests.filter(req => {
```

Comment:
nit: I think we can remove the `_` -- it was just to avoid the collision with the other `pendingRequests` variable.